sakura_core/extmodule/CBregexp.cpp

```diff
@@ -231,18 +231,18 @@ wchar_t* CBregexp::MakePattern( const wchar_t* szPattern, const wchar_t* szPatte
 {
 	using namespace WCODE;
 	static const wchar_t* szCRLF = CRLF;		//!< 復帰・改行
-	static const wchar_t szCR[] = {CR,0};				//!< 復帰
-	static const wchar_t szLF[] = {LF,0};				//!< 改行
+	static const wchar_t szCR[] = {CR,0};		//!< 復帰
+	static const wchar_t szLF[] = {LF,0};		//!< 改行
 	static const wchar_t BOT_SUBST[] = L"s/\\$(\\)*)$/([\\\\r\\\\n]+)\\$$1/k";	//!< 行末パターンの置換用パターン
 	int nLen;									//!< szPatternの長さ
 	BREGEXP_W*	sReg = NULL;					//!< コンパイル構造体
-	wchar_t szMsg[80] = L"";						//!< エラーメッセージ
-	wchar_t szAdd2[5] = L"";						//!< 行末あり置換の $数字 格納用
+	wchar_t szAdd2[5] = L"";					//!< 行末あり置換の $数字 格納用
 	int nParens = 0;							//!< 検索パターン(szPattern)中の括弧の数(行末時に使用)
-	wchar_t *szNPattern;							//!< 検索パターン
+	wchar_t *szNPattern;						//!< 検索パターン
 
 	nLen = CheckPattern( szPattern );
 	if( (m_ePatType & PAT_BOTTOM) != 0 ) {
+		wchar_t szMsg[80] = L"";			//!< エラーメッセージ
```

Comment:
```suggestion
		wchar_t szMsg[80] = {};			//!< エラーメッセージ
```

Reply:
@berryzplus fix it https://github.com/sakura-editor/sakura/pull/1860/commits/1fc1801a340c0d1ebb2b306d27b3d48c14ec2c17